hosting/technical/catalog_instructions.qmd

```diff
@@ -0,0 +1,41 @@
+# Creating Intake catalogs
+
+There are two versions of [intake](https://intake.readthedocs.io/en/latest/). Here, we focus on intake version 1. Version 2 has advanced features, but a [(partially) incompatible syntax](https://intake.readthedocs.io/en/latest/scope2.html#relationship-to-v1).
+
+## Simple entry
+
+As a minimal catalog, this yaml file would work:
+```yaml
+sources:
+  ngc3028:
+    args:
+      chunks: null
+      consolidated: true
+      urlpath: https://swift.dkrz.de/v1/dkrz_b381d76e-63d7-4aeb-96f0-dfd91e102d40/ngc3028_hp/ngc3028_P1D_5.zarr
+    driver: zarr
+```
+
+This can then be loaded and plotted via 
+```python
+import intake
+import healpy
+cat = intake.open_catalog("/path/to/catalog.yaml")
+ds = cat['ngc3028'].to_dask()
+tas_day0 = ds.tas.isel(time=0)
+healpy.mollview(tas_day0, nest=True, flip='geo')
+```
+
+## Working with netCDF files
+
+Examples for catalog entries with netCDF files can be found in [the nextGEMS catalog FESOM files](https://github.com/nextGEMS/catalog/blob/main/FESOM/IFS_4.4-FESOM_5-cycle3.yaml).
```

Comment:
Could you pull an example out and put it here as well? That would be helpful for the non-zarr folks. I assume driver: netCDF?